code/modules/admin/admin.dm

```diff
@@ -9,15 +9,19 @@ var/global/nologevent = 0
 			if(C.prefs && !(C.prefs.toggles & CHAT_NO_ADMINLOGS))
 				to_chat(C, msg)
 
-/proc/msg_admin_attack(var/text) //Toggleable Attack Messages
+/proc/msg_admin_attack(var/text, var/loglevel)
 	if(!nologevent)
 		var/rendered = "<span class=\"admin\"><span class=\"prefix\">ATTACK:</span> <span class=\"message\">[text]</span></span>"
 		for(var/client/C in admins)
 			if(R_ADMIN & C.holder.rights)
-				if(C.prefs.toggles & CHAT_ATTACKLOGS)
-					if(!istype(C, /mob/living))
-						var/msg = rendered
-						to_chat(C, msg)
+				if(istype(C, /mob/living))

```

Comment:
This can literally never happen. for(var/client/C automatically runs istype(C, /client) on every value in the list.